spec/models/comment_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should belong_to(:post) }
+  end
+end
```

Comment:
- [ ] Dear @Megagig, Just as mentioned by the previouse reviewer, here there should be a test that tests the update_post_comments_counter method. Please keep in mind we need to test all customer methods in our model. hence, we need to check creating a comment is incrementing a CommentsCounter by one.